exonum_client/module_manager.py

```diff
@@ -28,9 +28,10 @@ def import_main_module(module_name: str) -> Any:
         return module
 
     @staticmethod
-    def import_service_module(artifact_name: str, module_name: str) -> Any:
+    def import_service_module(artifact_name: str, artifact_version: str, module_name: str) -> Any:
```

Comment:
`import_service_module` in 'map_proof_builder.py' requires modification.

Reply:
Fixed already.